Fritz.StreamTools/Views/Followers/Docs_Goal.cshtml

```diff
@@ -42,6 +42,9 @@
 		<dt>current</dt>
 		<dd>A 'current follower count' to use for testing</dd>
 
+		<dt>fontName</dt>
+		<dd>An optional font name from Google Fonts</dd>
```

Comment:
May want to add a hyperlink to the Google Fonts page so folks can see what we're referencing